src/profiler.ts

```diff
@@ -186,9 +185,7 @@ export class Profiler extends common.ServiceObject {
       json: true,
     };
 
-    return this.request(options).then((result) => {
-      return;
-    });
+    await this.request(options);
```

Comment:
What's the intention around how errors in the request will be dealt with?

Reply:
Other profilers generally ignore errors which occur when attempting to upload the profile. I intend to do the same.

Reply:
Errors in collecting or uploading the profile will be ignored. 
I'll add this now, since it doesn't really complicate this too much.